src/components/transactions/TxSigners/index.tsx

```diff
@@ -138,41 +141,41 @@ export const TxSigners = ({
           </Box>
         </StepLabel>
       </StyledStep>
-      {!hideConfirmations &&
-        detailedExecutionInfo.confirmations.map(({ signer }) => getConfirmationStep(signer, signer.value))}
-      {detailedExecutionInfo.confirmations.length > 0 && (
-        <StyledStep $state={StepState.CONFIRMED}>
-          <StepLabel icon={<DotIcon />}>
-            <Link component="button" onClick={toggleHide} fontSize="medium">
-              {hideConfirmations ? 'Show all' : 'Hide all'}
-            </Link>
-          </StepLabel>
-        </StyledStep>
+      {!isPending && (
```

Comment:
I wouldn't hide them even tho the executioner might not be in the list. It's just how Safe works.

Reply:
Theoretically it's backend lag but in hindsight it's better to show what we have. I'll adjust it.